libbeat/metric/system/memory/memory.go

```diff
@@ -78,7 +78,8 @@ func GetSwap() (*SwapStat, error) {
 	// this can provoke too big values for used swap.
 	// Workaround this by assuming that all swap is free in that case.
 	if swap.Free > swap.Total || swap.Used > swap.Total {
-		logp.Debug("memory",
+		logger := logp.NewLogger("GetSwap")
+		logger.Debugf("memory",
```

Comment:
s/GetSwap/memory

don't change the "name" when creating `NewLogger`. The names are used as "debug selector" and changing names might be considered a breaking change.